src/main/java/com/google/sps/data/Event.java

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.Map;
+import java.util.Date;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.EmbeddedEntity;
+
+public class Event {
+
+  private long id;
```

Comment:
comments for each field, please. I'm confused why some ids are strings and some are ints and some are embedded entities.